### DISCUSSION TITLE
[help] Missing uri index method

### DISCUSSION THREAD

**Discussion by agungsugiarto** [spatie/laravel-route-discovery#23] [Q&A]
Controller

```php
<?php

namespace App\Http\Controllers\Legacy;

use App\Http\Controllers\Controller;

class ExampleController extends Controller
{
    public function index()
    {
    }
}
```

Output
```sh
 GET|HEAD   example ............. example › Legacy\ExampleController@index
```

Can we still added the uri index like this?
```sh
 GET|HEAD   example ............. example › Legacy\ExampleController@index
 GET|HEAD   example/index ............. example › Legacy\ExampleController@index
```

**Comment by SupianIDz** [comment]
I think this will contradict the laravel [resource controller](https://laravel.com/docs/9.x/controllers#actions-handled-by-resource-controller)

You can use route manually to add `index` to the route path.

**Comment by uyab** [comment]
You can override the uri using annotation:

https://spatie.be/docs/laravel-route-discovery/v1/discovering-routes-for-controllers/mapping-controllers-to-routes#customizing-the-url
